docs/data-ai/reference/triggers-configuration.md

```diff
@@ -1,4 +1,216 @@
-## Webhooks
+---
+title: "Trigger configuration"
+linkTitle: "Trigger configuration"
+weight: 60
+type: "docs"
+tags: ["data management", "trigger", "webhook"]
+description: "Detailed information about how to configure triggers and webhooks."
+date: "2025-05-05"
+---
+
+Triggers can alert you by email or webhook when the following events occur:
+
+- [machine telemetry data syncs from your local device to the Viam cloud](/manage/troubleshoot/alert/)
+- [data syncs from a machine](/data-ai/data/advanced/alert-data/)
+- [service detects a specified object or classifies a specified label](/data-ai/ai/alert/)
+
+## Trigger attributes
+
+Triggers support the following attributes:
+
+<!-- prettier-ignore -->
+| Name | Type | Required? | Description |
+| ---- | ---- | --------- | ----------- |
+| `name` | string | **Required** | The name of the trigger |
+| `event` |  object | **Required** | The trigger event object, which contains the following fields: <ul><li>`type`: The type of the event to trigger on. Options: `part_data_ingested`, `conditional_data_ingested`.</li><li>`data_types`: Required with `type` `part_data_ingested`. An array of data types that trigger the event. Options: `binary`, `tabular`, `file`, `unspecified`. </li><li> `conditional`: Required when `type` is `conditional_data_ingested`. </li></ul> |
```

Comment:
this should then also link to conditional attributes

Reply:
```suggestion
| `event` |  object | **Required** | The trigger event object, which contains the following fields: <ul><li>`type`: The type of the event to trigger on. Options: `part_data_ingested`, `conditional_data_ingested`.</li><li>`data_types`: Required with `type` `part_data_ingested`. An array of data types that trigger the event. Options: `binary`, `tabular`, `file`, `unspecified`. </li><li> `conditional`: Required when `type` is `conditional_data_ingested`. For more information about this field, see [Conditional attributes](/data-ai/reference/triggers-configuration/#conditional-attributes). </li></ul> |
```